app/templates/application.hbs

```diff
@@ -1,13 +1,21 @@
-{{page-title "My"}}
+{{page-title 'My | Real Dev Squad'}}
 
-<Navbar @firstName={{@model.firstName}} @profilePictureURL={{@model.profilePictureURL}} @signOutHandler={{this.signOutHandler}} />
+<Navbar
+  @firstName={{@model.firstName}}
+  @profilePictureURL={{@model.profilePictureURL}}
+  @signOutHandler={{this.signOutHandler}}
+/>
 
-<div class="main-container">
+<div class='main-container'>
   {{outlet}}
 </div>
 
-<footer class="footer">Contents of this website are deployed from this
-  <a href="https://github.com/Real-Dev-Squad/website-my" target="_blank" rel="noopener noreferrer">
+<footer class='footer'>Contents of this website are deployed from this
```

Comment:
Same as above I don't think this should be changed

Reply:
Corrected in the latest commit.